src/intro/hello-world.md

```diff
@@ -114,24 +213,70 @@ unsafe impl ExtensionLibrary for MyExtension {}
 
 There are multiple things going on here:
 
-1. Import the `prelude` module from the `godot` crate. This module contains the most common symbols in the gdext API.
+1. Place the [`prelude`][api-prelude] module from the [`godot`][api-godot] crate into scope.
+   This module contains the most common symbols in the gdext API.
 2. Define a struct called `MyExtension`. This is just a type tag without data or methods, you can name it however you like.
-3. Implement the `ExtensionLibrary` trait for our type, and mark it with the `#[gdextension]` attribute.
+3. Implement the [`ExtensionLibrary`][api-extensionlibrary] trait for our type, and mark it with the `#[gdextension]` attribute.
 
 The last point declares the actual GDExtension entry point, and the proc-macro attribute takes care of the low-level details.
 
 
-## Creating a Rust class
+#### "Does it load though?"
+
+If you want a quick look at having it load, you can start out by overloading the `on_level_init(InitLevel)` from the
+[`ExtensionLibrary`][api-extensionlibrary] trait like so:
+
+```rust
+use godot::prelude::*;
+
+struct MyExtension;
+
+#[gdextension]
+unsafe impl ExtensionLibrary for MyExtension {
+    fn on_level_init(level: InitLevel) {
+        godot_print!("Hello Library!");
+
+        godot_print!("Initialization level: {:?}", level);
+    }
+}
+```
+
+And then after `cargo build`, Godot should display the messages when you run Godot from the command line
+
+```log
+Godot/Tutorial/GodotProject
+$ godot4 -e
+Initialize GDExtension API for Rust: Godot Engine v4.2.beta1.official
+Hello Library!
+Initialization level: Core
+Godot Engine v4.2.beta1.official.b1371806a - https://godotengine.org
+Hello Library!
+Initialization level: Servers
+Vulkan API 1.3.246 - Forward+ - Using Vulkan Device #0: AMD - AMD Radeon Vega 8 Graphics (RADV RAVEN)
+
+Hello Library!
+Initialization level: Scene
+Hello Library!
+Initialization level: Editor
+```
+
+```admonish note
+These messages will most likely not show up unless you're either launching [Godot editor from your command line][godot-command-line] by using
+`godot -e` like in this example, or making your gdextension file visible for the first time in Godot while opened.
+```
```

Comment:
Instead of this note, we should probably explain the role of the `.godot/extension_list.cfg` file. Once that is there, this is not a problem and there's no probability involved anymore.